shell/platform/darwin/ios/ios_gl_render_target.mm

```diff
@@ -64,7 +64,8 @@
 IOSGLRenderTarget::~IOSGLRenderTarget() {
   [EAGLContext setCurrentContext:context_];
   FML_DCHECK(glGetError() == GL_NO_ERROR);
-
+ //fix platformview memory leak https://github.com/flutter/flutter/issues/24714
+  [EAGLContext setCurrentContext:context_.get()];
```

Comment:
Why are we setting this twice to the same value?  It's also not clear to me that this belongs in the same PR...

Reply:
hi, this is belongs other pr, i'll remove it next commit

Reply:
done